app/app_xdg.go

```diff
@@ -46,9 +46,7 @@ func (a *fyneApp) SendNotification(n *fyne.Notification) {
 }
 
 func rootConfigDir() string {
-	homeDir, _ := os.UserHomeDir()
-
-	desktopConfig := filepath.Join(homeDir, ".config")
+	desktopConfig, _ := os.UserConfigDir()
```

Comment:
Could we safely ignore the error here? After reading the doc, https://pkg.go.dev/os#UserConfigDir, this API may return an error if $HOME is not set. To me, this seems to be likely?

Reply:
The same error would have been send out before as well. It is not a change in behaviour with this PR.

Reply:
Indeed that's why I was wondering why the error was never reported. If there is an error, then the config path would be `$(pwd)/.config` because `homeDir` would be `""`?

Reply:
It should probably be checked but it might be outside of the scope of this PR. Not having $HOME set is also, to the best of my knowledge, very rare. The same error would have been sent out by os.UserHomeDir() as well.